src/ExcelProvider.Runtime/ExcelProvider.Runtime.fs

```diff
@@ -149,7 +149,7 @@ module internal ExcelAddressing =
 
         let minRow = ranges |> Seq.map (fun range -> range.StartRow) |> Seq.min
         let maxRow = ranges |> Seq.map (fun range -> range.EndRow) |> Seq.max
-        let rowCount = maxRow - minRow
+        let rowCount = (maxRow - minRow) + 1
```

Comment:
And here, if we had a minRow of 0, and maxRow of 2, we would calculate a rowCount of 2, when we want to reflect that there are 3 rows (0,1,2).

The change at 125 inflated the value of `maxRow` by 1; here, we see that value reduced by 1 again.

At this point, if I understand it correctly, an `Unbounded` `Range` would have the correct `RowCount`, while a `Bounded` `Range` would have a `RowCount` one less than intended.